.github/workflows/smoketest-preabort.sh

```diff
@@ -0,0 +1,18 @@
+echo 'List of processes:'
+ps aux 
+echo 'esp stacks'
+echo '=========='
+gdb --batch --quiet -ex "set interactive-mode off" -ex "thread apply all bt" -ex "quit" esp `pidof esp`
```

Comment:
It seems something wrong with "esp `pidof esp`".
In the regress\log\W20210322-124245-preAbort.log file: 
```
esp stacks
==========
/home/jsmith/hpccDebug/opt/HPCCSystems/bin/esp: No such file or directory.
Could not attach to process.  If your uid matches the uid of the target
process, check the setting of /proc/sys/kernel/yama/ptrace_scope, or try
again as the root user.  For more details, see /etc/sysctl.d/10-ptrace.conf
ptrace: Operation not permitted.
/home/runner/work/HPCC-Platform/HPCC-Platform/install/opt/HPCCSystems/testing/regress/9414: No such file or directory. 
```
Same problem with other stack traces.

Reply:
yes, it was fixed shortly after (removed erroneous path to exes)

Reply:
Should we re-run smoketest action again to check what it can catch if dfse2 problem occurred again?